oauth2client/contrib/appengine.py

```diff
@@ -89,11 +89,11 @@ def _safe_html(s):
     return cgi.escape(s, quote=1).replace("'", '&#39;')
 
 
-class InvalidClientSecretsError(Exception):
-    """The client_secrets.json file is malformed or missing required fields."""
+class Error(Exception):
+    """Base error for this module."""
```

Comment:
Yeah, it's overkill. I wouldn't create custom exceptions unless they are critically useful to users.